fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1523,6 +1523,11 @@ private void updateCatalogAfterCommitted(TransactionState transactionState, Data
         for (TableCommitInfo tableCommitInfo : transactionState.getIdToTableCommitInfos().values()) {
             long tableId = tableCommitInfo.getTableId();
             Table table = db.getTable(tableId);
+            if (table == null) {
+                LOG.warn("updateCatalogAfterCommitted table is null, table: {}, db: {}-{}",
```

Comment:
```suggestion
                LOG.warn("table is null, table: {}, db: {}-{}",
```
The log4j will print the method name itself, so there is no need to print the method name here.

Reply:
This is debug log, I have removed it.